src/devices_models/devices/common/range_constraint.jl

```diff
@@ -263,13 +263,18 @@ where limits in constraint_infos.
 """
 function add_reserve_range_constraints!(
     container::OptimizationContainer,
-    T::Type{<:PowerVariableLimitsConstraint},
+    T::Type{Z},
```

Comment:
Nit: can you order the variables correctly here, please. The last is the first and has the Z value.